src/lgr_advanced/lgr_editor/views/create.py

```diff
@@ -112,7 +113,7 @@ def form_valid(self, form):
 
         return super().form_valid(form)
 
-    def _handle_lgr_file(self, lgr_file, validating_repertoire, is_set, lgr_info_set):
+    def _handle_lgr_file(self, lgr_file, validating_repertoire: RzLgr, is_set, lgr_info_set):
```

Comment:
no, we would have other types of LGR as validating repertoire. You may use LgrModel type as there is a high probability that other validating repertoire inherit from it